packages/shared/components/FormInvite/FormInvite.tsx

```diff
@@ -64,13 +86,19 @@ export default function FormInvite(props: Props) {
       return;
     }
 
-    if (u2fEnabled) {
+    if (mfaType.value === 'u2f') {
       onSubmitWithU2f(password);
     } else {
       onSubmit(password, token);
     }
   }
 
+  function onSetMfaOption(option: MFAOption, validator: Validator) {
+    setToken('');
+    validator.reset();
+    setMfaType(option);
+  }
```

Comment:
i would also clear errors, lets say I was to set up a `u2f` key and it wasn't supported and errors out (safari) or any kind of error really, and I switch to `totp`, the u2f error dialog will still be there. I think any error should disappear on select new option